Documentation/tutorial/links_and_cross_references.md

```diff
@@ -0,0 +1,269 @@
+# Links and Cross References
+
+Markdown provides a [syntax](https://daringfireball.net/projects/markdown/syntax#link) to create hyperlinks.
+For example, the following syntax:
+```markdown
+[bing](http://www.bing.com)
+```
+Will render to:
+
+```html
+<a href="http://www.bing.com">bing</a>
+```
+
+Here the url in the link could be either absolute url pointing to another website (`www.bing.com` in the above example),
+or a relative url pointing to a local resource on the same server (for example, `/about/`).
```

Comment:
`/about/` is considered as an absolute path in `docfx`, is the description here confusing?